styles.css

```diff
@@ -0,0 +1,398 @@
+html,
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+input,
+button,
+ul,
+li,
+a,
+p {
+  background: transparent;
+  list-style: none;
+  color: inherit;
+  font: inherit;
+  text-align: inherit;
+  text-decoration: inherit;
+  margin: 0;
+  border: 0;
+  padding: 0;
+}
+
+/* Global Setting */
+
+/* Import custom font Cocogoose for use in headers */
+@font-face {
+  font-family: Cocogoose;
+  src: url("../assets/fonts/cocogoose/Cocogoose Pro-trial.ttf");
+}
+
+/* Set default font for entire body to Lato */
+body {
+  font-family: "Lato", sans-serif;
+}
+
+/* Headers use custom font Cocogoose, with font-weight set to bold and background-size set to 100% */
+h1 {
+  font-size: 5vw;
+  font-family: "Cocogoose", sans-serif;
+  font-weight: bold;
+  background-size: 100%;
+  color: transparent;
+}
+
+h2 {
+  font-family: "Cocogoose", sans-serif;
+  font-size: 5vw;
+}
+
+h3 {
+  font-size: 5vw;
+}
+
+/* Set width of all images to 100px */
+img {
+  width: 100px;
+}
+
+/* Set font-size for class 'tickets-title' to 20px */
+.tickets-title {
+  font-size: 20px;
+}
+
+/* Change cursor to pointer when hovering over links and change color when active */
+a {
+  cursor: pointer;
+}
+
+a:hover {
+  color: #ec5242;
+}
+
+a:active {
+  color: #ec5242;
+}
+
+/* Set color and font-weight for all paragraphs */
+p {
+  color: #272a31;
+  font-weight: 100;
+}
+
+/* The row class is used to set the left and right margins to 0 */
+.row {
+  margin-left: 0; /* Remove any left margin */
+  margin-right: 0; /* Remove any right margin */
+}
+
+/* The cta class is used to set the font size for call-to-action elements */
+.cta {
+  font-size: 1.2rem; /* Increase the font size for better visibility */
```

Comment:
For best practices, with all the comments your codebase looks ugly, your code is well written and it can be understood.

It would be nice to remove unnecessary comments.